Deck_Shuffler/Deck_Shuffler/Deck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Deck
+    {
+        public List<Card> Cards = new List<Card>();
+        
+        public Deck()
+        {
+            string[] suits = new string[] { "Hearts", "Diamonds", "Clubs", "Spades" };
```

Comment:
Most devs would write `var suits =` here. Seeing two string[] would be considered unsightly.